sql-cli/sql_cli/sql_directory_parser.py

```diff
@@ -21,6 +22,7 @@ def __init__(self, root_directory: Path, path: Path, target_directory: Path) ->
         self.root_directory = root_directory
         self.path = path
         self.target_directory = target_directory
+        self.raw_content = open(self.path).read()
```

Comment:
```suggestion
        self.raw_content = self.path.read_text()
```
[Path.read_text](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text)